site/docs/guides/contributor/release-process/egeria-react-ui-release.md

```diff
@@ -0,0 +1,57 @@
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+
+
+??? success "1. Agree schedule"
+    --8<-- "docs/guides/contributor/release-steps/agree-schedule.md"
+??? success "2. Track remaining issues and PRs" 
+    --8<-- "docs/guides/contributor/release-steps/track-outstanding.md"
+??? success "3. Create branch"
+    - Checkout master `git checkout main`
+    - Ensure local update `git pull upstream main`
+    - Create branch `git branch release-x.y`
+    - Push to upstream `git push upstream release-x.y`
+
+??? success "4. Update main from `x.y-rc` to `x.z-rc`"
+    - `git checkout main`
+    - `git pull upstream main`
+    - Edit cra-client/package.json `x.y-rc` with the next version `x.z-rc`, e.g. change `3.1-rc` to `3.2-rc`.
+    - Edit cra-server/package.json `x.y-rc` with the next version `x.z-rc`, e.g. change `3.1-rc` to `3.2-rc`.
+    - Commit
+    - Create a PR, have reviewed / approved and merged as usual - aim to do this as quickly as reasonable so that there is no potential for version clash
+
+??? success "5. Test, merge any remaining required changes into branch"
+    - Run appropriate tests for the release. For example, in addition to automated tests: check notebooks, run the CTS and check for compliance, check the user interface.
+    - Raise issues for any changes required as usual
+    - Note that approval is required for changes going into a release branch
+    - PR builds are run as usual; however, merge builds, Sonar, etc do not run
+    - To backport changes from `main`, first wait until the PR is merged into `main`, then use `git cherrypick -s <commithash>` to apply to `release-x.y`, then push as usual.
+    - In some cases a merge commit will need to be made using `git cherrypick -s -m 1 <commithash>`
+    - If code has diverged significantly a manual recode may be easiest
+
+??? success "6. Update branch's release version from `x.y-rc` to `x.y`"
+    - Aim to make this change when the code appears to be ready to ship apart from final tests in order to avoid version confusion
+    - Edit cra-client/package.json `x.y-rc` with the next version `x.y`, e.g. change `3.2-rc` to `3.2`.
+    - Edit cra-server/package.json `x.y-rc` with the next version `x.y`, e.g. change `3.2-rc` to `3.2`.
+    - Commit, and do not make any other changes.
+    - Create a PR, have reviewed / approved and merged as usual
+
+??? success "7. Create a release in GitHub"
+    - [Create the GitHub release :material-github:](https://github.com/odpi/egeria-react-ui/releases){ target=gh }. Use `Vx.y` as the tag, and ensure the correct branch is set for the target, i.e. `egeria-release-x.y`
+    - Fill in the release notes using a title of `Release x.y` and [copy the notes from the appropriate release notes](/release-notes/overview)
```

Comment:
see comments on egeria for the auto gen release notes

Reply:
See https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
<img width="1291" alt="Screenshot 2022-02-08 at 13 45 56" src="https://user-images.githubusercontent.com/7292002/152999660-1e38fd12-ca6e-4c62-b4fe-c50a34489576.png">

From that full list generated, unfortunately only part of it is correct, so reviewing it I would have edited it to this point -- looking for the last release increment ie:
```
* odpi/egeria#5929 update main to 3.5.0-rc.0 by @planetf1 in https://github.com/odpi/egeria-react-ui/pull/295
* #288 refresh the server list after an add by @davidradl in https://github.com/odpi/egeria-react-ui/pull/309
* #310 update dependancies by @davidradl in https://github.com/odpi/egeria-react-ui/pull/312
* #318 point readme to react ui docs in egeria-docs by @davidradl in https://github.com/odpi/egeria-react-ui/pull/319
* #318 fix readme by @davidradl in https://github.com/odpi/egeria-react-ui/pull/320
* #8 clean up old docs by @davidradl in https://github.com/odpi/egeria-react-ui/pull/323
* git325 fix quick terms and add large description by @davidradl in https://github.com/odpi/egeria-react-ui/pull/331
* #8 fix link and typo by @davidradl in https://github.com/odpi/egeria-react-ui/pull/324
* #236 add community profile server author support  by @davidradl in https://github.com/odpi/egeria-react-ui/pull/262
* #339 add access service options  by @davidradl in https://github.com/odpi/egeria-react-ui/pull/340
* #342 resolve the critcal npm audit vulnerability by @davidradl in https://github.com/odpi/egeria-react-ui/pull/343
* #344 document findings by @davidradl in https://github.com/odpi/egeria-react-ui/pull/346
* #341 clean up access service options by @davidradl in https://github.com/odpi/egeria-react-ui/pull/347
* #344 correct typos and remove mkdocs workflow by @davidradl in https://github.com/odpi/egeria-react-ui/pull/350
* git352 update packages in main by @davidradl in https://github.com/odpi/egeria-react-ui/pull/354

## New Contributors
* @grahamwallis made their first contribution in https://github.com/odpi/egeria-react-ui/pull/12
* @sarbull made their first contribution in https://github.com/odpi/egeria-react-ui/pull/57
* @dependabot made their first contribution in https://github.com/odpi/egeria-react-ui/pull/105
* @tcnt made their first contribution in https://github.com/odpi/egeria-react-ui/pull/294

**Full Changelog**: https://github.com/odpi/egeria-react-ui/commits/3.5.1
```

BUT IN ADDITION see the last two sections., - they also need editing to reflect where the last release left off. That needs some manual thought currently -- github is getting the base of the compare wrong

I posted in the github discussion and apparently the issue was escalated - but it still seems broken, at least for the UI -> https://github.com/github/feedback/discussions/5975#discussioncomment-1580711